### PR TITLE
fix the emoji issue

### PR DESCRIPTION
**Pull request by fengsh27**
### Abstract
The biochatter-next only support the following emoji code:
1. "1f468-200d-1f52c" or,
2.  "👨‍🔬"

### CODE REVIEW

File: config/biochatter-next.yaml [hunk 1]
```diff
@@ -69,14 +69,14 @@ Text:
           content: "You can receive additional information from multiple tools: A knowledge graph containing simulated patient information about their clinical history, somatic mutations and copy numbers of specific genes, the biological functions and druggability information of the affected genes; a searchable vector database with embeddings of relevant scientific literature on ovarian cancer; and a web API that can return information on cancer genes (only on BRAF, TP53 and ROS1) and treatment implications (OncoKB). The tools are used, one at a time, at the command of the user."
           date: ""
       modelConfig:
-        model: gpt-3.5-turbo
+        model: gpt-4o-mini
```
Comment (slobentanzer) [inline config/biochatter-next.yaml:72]: @fengsh27 should we do this here?

**Review by slobentanzer — CHANGES_REQUESTED** [review]
we should agree on the default model to be used. I didn't see gpt-4o-mini in the list of models selector, does this work like that?

File: config/biochatter-next.yaml (hunk 1 above)
Comment (fengsh27) [reply config/biochatter-next.yaml:72]: Currently, in biochatter-next,  gpt model is not customizable. As biochatter-server determins what gpt model we are using and we don't want to expose this to users,  we don't pass the corresponding arguments to biochatter-server. I'll modify biochatter-next to make it unchangable

Comment (fengsh27) [reply config/biochatter-next.yaml:72]: I've raised a PR in biochatter-next to address the issue

**Pull request merged**